client/webserver/site/src/js/markets.js

```diff
@@ -579,21 +579,22 @@ export default class MarketsPage extends BasePage {
    * in response to a new market subscription. The data received will contain
    * the entire order book.
    */
-  handleBookRoute (data) {
-    app.log('book', 'handleOrderBook:', data)
+  handleBookRoute (note) {
+    app.log('book', 'handleBookRoute:', note)
+    const mktBook = note.payload
     const market = this.market
     const page = this.page
     const host = market.dex.host
     const [b, q] = [market.baseCfg, market.quoteCfg]
-    if (data.base !== b.id || data.quote !== q.id) return
-    this.handleBook(data)
+    if (mktBook.base !== b.id || mktBook.quote !== q.id) return
+    this.handleBook(mktBook)
     page.marketLoader.classList.add('d-none')
     this.marketList.select(host, b.id, q.id)
 
     State.store(lastMarketKey, {
-      host: data.host,
-      base: data.base,
-      quote: data.quote
+      host: note.host,
+      base: mktBook.base,
+      quote: mktBook.quote
```

Comment:
These changes are just to accommodate the change from `marketResponse` to `BookUpdate`.